usmqe/ceph/commands.py

```diff
@@ -10,6 +13,11 @@
 
 
 LOGGER = pytest.get_logger('usmceph.commands', module=True)
+usmqe.usmssh.KEYFILE = "~/.ssh/id_rsa"
+try:
+    usmqe.usmssh.KEYFILE = pytest.config.getini("usm_ssh_keyfile")
+except ValueError:
+    pass
```

Comment:
Paramiko(which is used in this case) doesn't use ssh configuration. That's why there is key as parameter. I've tried it with ssh configuration before I've made this change.